app/controllers/conversations/messages_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Conversations
+  class MessagesController < ApplicationController
+    def create
+      @conversation = find_conversation
+      @message_form = MessageForm.new(
+        message_params.merge(conversation: @conversation)
+      )
+      @message_form.save
+
+      if @message_form.persisted?
+        redirect_to @conversation
+      else
+        render "conversations/show"
+      end
+    end
+
+  private
+
+    def message_params
+      params
+        .require(:message)
+        .permit(:body, :hat_id, :mod_note)
```

Comment:
Only moderators should be permitted to set the `mod_note` param. There's an example of this in the old `MessagesController`.

Reply:
I believe we have that logic here: https://github.com/lobsters/lobsters/pull/644/files#diff-3ac3bb9392913059755d80f258ad66ffR37

Care to confirm?